### PR TITLE
Remove deprecated commit_required parameter from JetPack

### PR DESCRIPTION
**Pull request by Rachit7194**
I have removed `commit_required `parameter with `is_commit_required` and `is_reboot_required `as per the given condition respectively. Guys @rpioso @digambar15  @cdearborn please review it.

### CODE REVIEW

File: src/pilot/config_idrac.py [hunk 1]
```diff
@@ -142,9 +142,9 @@ def configure_uefi_nics_boot_settings(drac_client, pxe_nic_id):
 
             provisioning_mac = nic.mac_address.lower()
             response = drac_client.set_bios_settings(settings)
-            reboot_required = response['commit_required']
+            reboot_required = response['is_reboot_required']
 
-            if response['commit_required']:
+            if response['is_commit_required']:
```
Comment (cdearborn) [inline src/pilot/config_idrac.py]: Would you mind changing the above line to:
if reboot_required:

Comment (cdearborn) [reply src/pilot/config_idrac.py]: I'm sorry, please ignore the above comment.  Line 147 is correct as it is.

**Comment by cdearborn** [comment]
Looks good!  Please post a comment when testing of the assign_role change is complete.

**Comment by Rachit7194** [comment]
I have tested the assign_role successfully and we are good to go.

File: src/pilot/config_idrac.py [hunk 2]
```diff
@@ -211,13 +211,13 @@ def config_boot_mode(drac_client, ip_service_tag, node, boot_mode):
     response = drac_client.set_bios_settings(settings)
 
     job_id = None
-    if response['commit_required']:
+    if response['is_commit_required']:
         job_id = drac_client.commit_pending_bios_changes(reboot=False,
                                                          start_time=None)
 
     # Note that "commit_required" is actually "reboot_required" under the
```
Comment (dpaterson) [inline src/pilot/config_idrac.py]: This comment is confusing now, should it be removed?

Comment (cdearborn) [reply src/pilot/config_idrac.py]: yes.  since commit_required has been removed, we should remove this comment too.

**Pull request merged**